### ISSUE TITLE
flatpak update fails, Non-prefixed filename org.gpodder.service in app org.gpodder.gpodder, removing.

### ISSUE THREAD

**Issue by Mikaela** [flathub/org.gpodder.gpodder#5]
```
# LANG=C flatpak update
Looking for updates...
Updating: org.gpodder.gpodder/x86_64/stable from flathub
[####################] 1 metadata, 0 content objects fetched; 569 B transferred in 0 seconds
(flatpak update:15618): flatpak-WARNING **: 21:14:57.133: Non-prefixed filename org.gpodder.service in app org.gpodder.gpodder, removing.

Error: Failed to update org.gpodder.gpodder/x86_64/stable: No such file or directory
error: One or more operations failed
```

I have been seeing this for a day or two, is this an issue with this flatpak or should I do something special to fix this?

**Comment by ndarilek** [comment]
Just pushed a commit that might fix this. Anyone know why there was an update listed to begin with? Looks like this repo hasn't changed since March 29, so not sure why a new update appeared and things started breaking.

In any case, I think I renamed the dbus service, though I do a agree that this shouldn't block *all* updates.

**Comment by Mikaela** [comment]
Thanks, I can confirm that it works now.

```
# LANG=C flatpak update                     
Looking for updates...
Updating: org.gpodder.gpodder/x86_64/stable from flathub
[####################] 1 delta parts, 1 loose fetched; 7 KiB transferred in 1 seconds
Now at 5cc1cc8f69b9.
Updating: org.gpodder.gpodder.Locale/x86_64/stable from flathub
[####################] 3 metadata, 0 content objects fetched; 2 KiB transferred in 0 seconds
Now at ce86ae161f4d.
Updating appstream data for remote flathub
```

I don't think I understand this issue enough to open a new one to flatpak/flatpak. I tried asking for opinion about this on #flatpak earlier, but I think no one saw it.

**Comment by TingPing** [comment]
This was rebuilt again, please let me know if it still fails.

**Comment by TingPing** [comment]
What `flatpak` version? That is where the issue would be.

**Comment by Mikaela** [comment]
```# LANG=C flatpak update
Looking for updates...
Updating: org.gpodder.gpodder/x86_64/stable from flathub
[####################] 1 metadata, 0 content objects fetched; 569 B transferred in 0 seconds
(flatpak update:6555): flatpak-WARNING **: 10:50:25.753: Non-prefixed filename org.gpodder.service in app org.gpodder.gpodder, removing.

Error: Failed to update org.gpodder.gpodder/x86_64/stable: No such file or directory
error: One or more operations failed
```

`Flatpak 0.11.3`, I am using Ubuntu MATE 18.04 LTS.

**Comment by Mikaela** [comment]
Installing `Flatpak 0.11.6` from the PPA at [flatpak Ubuntu setup instructions](https://flatpak.org/setup/Ubuntu/) fixes this for me.

I am not sure whether I should close this issue again, so I am leaving it for you. I am thinking that on the other hand open issue is easier to find and I think it's likely for someone to open a duplicate, but I guess this is more of flatpak/flatpak or Ubuntu issue for shipping older version of flatpak?

**Comment by TingPing** [comment]
I suggest opening an issue against flatpak/flatpak, noting that it is fixed in 0.11.x and broken in 0.10.4, I'm sure we can get that fix backported.

**Comment by nedrichards** [comment]
Noted, this isn't a gpodder bug specifically and should be opened upstream if still relevant.
